features/fields-mapping-Persoon.csv

```diff
@@ -1,21 +1,73 @@
-veld,volledig pad
+fields pad,volledig pad
```

Comment:
Hier mist inOnderzoek.datumIngangOnderzoek

Reply:
goed gevonden! probleem zit in de specificaties:
```
    PersoonInOnderzoek:
      description: |
        Geeft aan welke gegevens van de persoon in onderzoek zijn. Zie de [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.1.0/features/in_onderzoek.feature).
      allOf:
        - $ref: '#/components/schemas/InOnderzoek'
      type: object
      properties:
        burgerservicenummer:
          type: boolean
        geslacht:
          type: boolean
        leeftijd:
          type: boolean
        gemeenteVanInschrijving:
          type: boolean
        datumInschrijvingInGemeente:
          type: boolean
```
De uitbreiding (alles na allOf en $ref) had na een streepje moeten staan en iets naar rechts. Blijkbaar kunnen andere generatoren hier tegen, maar ik had daar niet op gerekend

Reply:
ik heb het aangepast in de specificaties. inOnderzoek.datumIngangOnderzoek zit nu wel in de fields-Persoon.csv